tests/functional/c/class_attributes.py

```diff
@@ -1,7 +1,7 @@
 """Test that valid class attribute doesn't trigger errors"""
-__revision__ = 'sponge bob'
```

Comment:
All right, someone had fun here 😄

Reply:
It feels a real shame to remove such a humorous line :D